frontend/packages/console-app/console-extensions.json

```diff
@@ -472,7 +472,12 @@
       "id": "projects",
       "insertAfter": "dashboards",
       "name": "%console-app~Projects%",
-      "resource": "projects"
+      "startsWith": ["projects"],
```

Comment:
The more I think about this -- especially given my other note I just made on how I thought this field worked... I am hesitant to agree this is needed. I'd think we should be able to infer this value from the model 🤔 eg a `ProjectsModel.plural` is all this is... and you describe the model effectively (with this change).

Can you look into removing the `startsWith` as much as possible? I think it's an "extra" feature for flexibility of "sub pages" and not intended to be a mainstay of the way the navigation works.

Reply:
Could we address this as a follow-on tech debt issue? I only added these startsWith props so that we retain active nav items for old links, and I'm not sure it's worth the work of implementing a special case to match on model.plural.

Reply:
Personally I'd deal with it where you introduced it, but if you want to push it for some reason, it should remain a priority to fix it and not slip through the cracks. You/OCP team can determine how you want to split up the work, I'm just noting my concerns 🙂 

I view this as an issue with the extension if we are not effectively using it properly when we consume the data. `startsWith` is optional... if you do not work without it, I'd call this a bug -- other plugins will struggle through this pain too.